### DISCUSSION TITLE
关于0.0.36启动的时候，鼠标卡顿的问题

### DISCUSSION THREAD

**Discussion by WENSHAO521** [nextai-translator/nextai-translator#555] [General]
关于0.0.36启动的时候，鼠标卡顿的问题是哪里的问题，如果是部分代码的问题，我希望可以修复或者改善一下。

辛苦了♥，开发者

**Comment by StandfordYang** [comment]
我也会，但是一般都直接右键管理员运行

**Comment by WENSHAO521** [comment]
感觉0.0.37修復了這個問題

MikeYang ***@***.***> 于 2023年4月3日周一 19:44写道：
